pages/task/list/content/index.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
   'no-tasks': {
     outstanding: 'You have no outstanding tasks',
     inProgress: 'You have no tasks in progress',
-    completed: 'You have no completed tasks',
+    completed: 'You have no completed tasks test',
```

Comment:
Please revert this, its not related to this work.
```suggestion
    completed: 'You have no completed tasks',
```